tests/react/vanilla-utils/freezeAtom.test.tsx

```diff
@@ -35,6 +35,64 @@ it('freezeAtom basic test', async () => {
   await screen.findByText('isFrozen: true')
 })
 
+it('freezeAtom handles null correctly', async () => {
+  const nullAtom = atom<null, [null], void>(null, (_get, set, _arg) => {
```

Comment:
Does this work?
```suggestion
  const nullAtom = atom(null, (_get, set, _arg) => {
```

Reply:
Oh, I see. because the arg is required.
Let's do this then:
```suggestion
  const nullAtom = atom(null, (_get, set, _arg: null) => {
```